kratos/factories/base_factory.h

```diff
@@ -0,0 +1,236 @@
+//    |  /           |
```

Comment:
can you please change the file name to match the class name

Reply:
OK